docset/winserver2025-ps/smbshare/Get-SmbShare.md

```diff
@@ -16,28 +16,33 @@ Retrieves the SMB shares on the computer.
 ## SYNTAX
 
 ```
-Get-SmbShare [[-Name] <String[]>] [[-ScopeName] <String[]>] [-Scoped <Boolean[]>] [-Special <Boolean[]>]
- [-ContinuouslyAvailable <Boolean[]>] [-ShareState <ShareState[]>]
+Get-SmbShare [[-Name] <String[]>] [[-ScopeName] <String[]>] [-Scoped <Boolean[]>]
+ [-Special <Boolean[]>] [-ContinuouslyAvailable <Boolean[]>] [-ShareState <ShareState[]>]
  [-FolderEnumerationMode <FolderEnumerationMode[]>] [-CachingMode <CachingMode[]>]
- [-LeasingMode <LeasingMode[]>] [-ConcurrentUserLimit <UInt32[]>] [-AvailabilityType <AvailabilityType[]>]
- [-CaTimeout <UInt32[]>] [-EncryptData <Boolean[]>] [-CompressData <Boolean[]>] [-IncludeHidden]
+ [-LeasingMode <LeasingMode[]>] [-DirectoryHandleLeasing <Boolean[]>]
+ [-ConcurrentUserLimit <UInt32[]>] [-AvailabilityType <AvailabilityType[]>] [-CaTimeout <UInt32[]>]
+ [-EncryptData <Boolean[]>] [-CompressData <Boolean[]>] [-IsolatedTransport <Boolean[]>]
+ [-QoSFlowScope <QoSFlowScope[]>] [-QoSPolicyId <String[]>] [-IncludeHidden]
  [-SmbInstance <SmbInstance>] [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob]
  [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Get-SmbShare** cmdlet retrieves objects that represent the Server Message Block (SMB) shares being displayed by the computer.
+
+The `Get-SmbShare` cmdlet retrieves objects that represent the Server Message Block (SMB) shares
+being displayed by the computer.
 
 ## EXAMPLES
 
 ### Example 1: Get SMB shares on a local computer
-```
-PS C:\>Get-SMBShare
+
+```powershell
+PS C:\\>Get-SMBShare
```

Comment:
All of these double \\ are wrong (unless this is some escape character for markup?)

Get-smbshare

Name   ScopeName Path       Description
----   --------- ----       -----------
ADMIN$ *         C:\WINDOWS Remote Admin
C$     *         C:\        Default share
IPC$   *                    Remote IPC